### ISSUE TITLE
cannot replicate figure 4

### ISSUE THREAD

**Issue by shabbychef** [damienchallet/sharpeRratio#1]
Hi;

I am having troubles replicating figure 4 from v6 of the paper as found on [arxiv](https://arxiv.org/pdf/1505.01333.pdf). Rather than estimate the standard errors, as you apparently have done in the paper, I compute the MSE of the estimators empirically. Here is my equivalent plot for a t(3) and Gaussian distribution:

![relative_efficiency](https://user-images.githubusercontent.com/1456159/65935745-0d286f80-e3cf-11e9-9b1b-5e1dc17cf91c.png)

I believe the apparent problems for large c/sigma are due to the range of the spline; these do not concern me much as those seem to occur for very large Signal Noise values, which you are unlikely to encounter in the real world. However, I get nothing like the factor of 5x relative efficiency seen in figure 4's left facet. (My empirical efficiency for the Gaussian case is also rather flat in comparison.)

To double check, I have tried to reduce my code to the bare minimum, checking the c/sigma=0.1 case for n=50, using the package from this repo. I get a relative efficiency of around 1.5, consistent with my sims, but nowhere near 5.
```r
library(sharpeRratio)
nu <- 3  # tail exponent = 4
trescal <- sqrt((nu-2)/nu)

# unit variance:
set.seed(4123)
sd(trescal * rt(1e6,df=nu))

mu <- 0.1  # true SNR
n <- 50  # days
set.seed(1234)
sims <- replicate(1000,{
  x <- mu + trescal * rt(n,df=nu)
  sr <- mean(x) / sd(x)   # moment based

  # drawdown
  sim <- estimateSNR(x)
  dd <- sim$SNR
  c(sr,dd)
})
# you can check the bias:
biases <- rowMeans(sims) - mu
mse <- function(x) { mean(x^2) }
mses <- apply(sims - mu,1,FUN=mse)
rel_eff <- (mses[1]/mses[2])
```
This yields
`
[1] 1.489
`

**Comment by damienchallet** [comment]
Indeed, this comes both from the spline and from the factor 8/3 which
shrinks the estimate, as you have previously noted, and thus the standard
deviation as well. Because the version of the package on github has still a
1-10% bias, the apparent MSE is larger than it could be. I am writing up
both an erratum and an improved package. Thanks for your comments.

**Comment by shabbychef** [comment]
Ah, I see. I had interpreted the plot to be based entirely on theoretical standard error, not empirical, so did not think it was affected by implementation issues.
